src/js/components/Navigation/HeaderBarProfilePopUp.jsx

```diff
@@ -1,40 +1,47 @@
-import { Button } from '@mui/material';
-import withStyles from '@mui/styles/withStyles';
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import styled from 'styled-components';
-import { isAndroidSizeWide, isAndroidSizeXL, restoreStylesAfterCordovaKeyboard } from '../../common/utils/cordovaUtils';
-import { isCordova, isWebApp } from '../../common/utils/isCordovaOrWebApp';
-import { renderLog } from '../../common/utils/logging';
-import webAppConfig from '../../config';
-import DeviceURLField from '../SignIn/DeviceURLField';
+import { Button } from "@mui/material";
```

Comment:
Please don't use the auto-formatting of entire pages. Please roll back all of these changes, except for the changes you made to comment out lines 135-161 -- those changes are correct.